parameter/LogarithmicParameterAdaptation.cpp

```diff
@@ -78,10 +78,10 @@ bool CLogarithmicParameterAdaptation::fromXml(const CXmlElement& xmlElement,
 }
 
 
-int64_t CLogarithmicParameterAdaptation::fromUserValue(double dValue) const
+int64_t CLogarithmicParameterAdaptation::fromUserValue(double value) const
 {
-    return fmax(round(base::fromUserValue(log(dValue) / log(_dLogarithmBase))),
-                        _dFloorValue);
+    return int64_t(fmax(round(double(base::fromUserValue(log(value) / log(_dLogarithmBase)))),
```

Comment:
- WTF ? fromUserValue returns a uint64_t , why then casting it to double to round it ? 
 - fmap => max<int64_t> ?